file-formats/dsfi/type/zif_aff_dsfi_v1.intf.abap

```diff
@@ -2,18 +2,36 @@ INTERFACE zif_aff_dsfi_v1
   PUBLIC.
 
 
-  "! <p class="shorttext">Engine</p>
-  "! Engine
-  "! $values {@link zif_aff_dsfi_v1.data:co_engine}
-  TYPES: ty_engine TYPE zif_aff_types_v1=>ty_object_name_30.
+  TYPES:
+    "! <p class="shorttext">Engine</p>
+    "! Engine
+    "! $values {@link zif_aff_dsfi_v1.data:co_engine}
+    "! $default {@link zif_aff_dsfi_v1.data:co_engine.analytical_engine}
+    ty_engine TYPE zif_aff_types_v1=>ty_object_name_30.
 
   CONSTANTS:
     BEGIN OF co_engine,
       "! <p class="shorttext">Analytical Engine</p>
       "! Analytical engine
       analytical_engine TYPE ty_engine VALUE 'ANA',
+      "! <p class="shorttext">SQL Engine</p>
+      "! SQL engine
+      sql_engine TYPE ty_engine VALUE 'SQL',
     END OF co_engine.
 
+  TYPES:
+    BEGIN OF ty_sql_properties,
+      "! <p class="shorttext">AMDP Reference</p>
+      "! The name of the AMDP implementation reference
+      "! $required
+      amdp_reference               TYPE c LENGTH 100,
+      "! <p class="shorttext">Auto Exposed in SQL Services</p>
```

Comment:
Thank you for changing the field to `auto_exposed_in_sql_services`. I think this is a great improvement.
As mentioned by @fvalves, we cannot use "automatically" for the field, because we exceed ABAP's limitation of 30 characters for fields. However, in this case, we could keep the "Automatically" in the title.

@fvalves Thanks for keeping it in sync as I explained you. I guess, here it is worth to use the full term. 

If you agree, I can perform required changes in this PR.

```suggestion
      "! <p class="shorttext">Automatically Exposed in SQL Services</p>
```

Changing this requires to change `dsfi-v1.json`, too.